### PR TITLE
Changed readme install description.

### LINKED ISSUE #9: rhof-cli is not in npm registry
the command `npm i -g rhof-cli` no longer works with npm
probably because the package name was changed in npm registry

on querying the npm website it suggests `npm i @gh-conf/rhof`

seems like the documentation needs an update.
I can create a PR modifying and correcting the README.

### PR DESCRIPTION
**Pull request by theNaut1lus**
readme installation description now matches with npm website.

Fixes #9 
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ CLI tool to generate github contributors hall of fame for github readme
 ## Install
 
 ```shell
-$ npm i -g rhof-cli
+$npm -g i @gh-conf/rhof
```
Comment (arshadkazmi42) [inline README.md]: it should be 
```shell
$ npm i -g rhof
```
Name of the package is `rhof`
`@gh-conf/rhof` is the base package used in this library

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Need some changes

File: README.md (hunk 1 above)
Comment (theNaut1lus) [reply README.md]: It specifies on npm to write`@gh-conf/rhof`

Made the changes nevertheless

Comment (arshadkazmi42) [reply README.md]: This is the correct npm url
https://www.npmjs.com/package/rhof for this package

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ CLI tool to generate github contributors hall of fame for github readme
 ## Install
 
 ```shell
-$ npm i -g rhof-cli
+$npm -g i rhof
```
Comment (arshadkazmi42) [inline README.md]: let's also add a space after `$` to make it clear, that $ is for command line. Its not part of the install command

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for all the contributions you have been doing.
Just a small change and its ready to merge

File: README.md (hunk 1 above)
Comment (theNaut1lus) [reply README.md]: Then it is showing the wrong install code on npm as well.

Comment (arshadkazmi42) [reply README.md]: Can you share what is the error you are getting on running this 
`npm i -g rhof`
?

**Review by arshadkazmi42 — APPROVED** [review]
:+1:

**Comment by arshadkazmi42** [comment]
@co16353sidak you can send PR to this repo also with updated contributors Hall of Fame

**Pull request merged**